cmd/neofs-cli/modules/container.go

```diff
@@ -114,13 +116,13 @@ var listContainersCmd = &cobra.Command{
 		var oid *owner.ID
 
 		key, err := getKey()
-		exitOnErr(cmd, err)
+		common.ExitOnErr(cmd, "", err)
```

Comment:
IMO it'd be much more convenient to have `common.ExitOnErr` and `common.ExitOnErrf` funcs in order to not pass `""` in a lot of places.

Reply:
Let's try create other modules and see if we need anything. Internal package will be revisited at the end of the refactoring.